sycl/plugins/level_zero/usm_allocator.cpp

```diff
@@ -643,6 +653,17 @@ void *Slab::getEnd() const {
 
 bool Slab::hasAvail() { return NumAllocated != getNumChunks(); }
 
+void Bucket::decrementPool(bool &FromPool) {
+  // If a slab was available in the pool then note that the current pooled
```

Comment:
Nitpick. Comment can be moved above the function?

Reply:
Done